### PR TITLE
Commented and made some code easier to read

### PR DESCRIPTION
**Pull request by bseto**
Changed the WiringPi repo because travis initially got this error:

```
Cloning into 'wiringPi'...
fatal: unable to connect to git.drogon.net:
git.drogon.net[0: 188.246.205.22]: errno=Connection refused
git.drogon.net[1: 2a03:9800:10:7b::2]: errno=Cannot assign requested address

Installer/MainInstaller.sh: 99: cd: can't cd to wiringPi
```

On my local computer I am also getting the same error:
```
byron@byron-570X Documents $git clone git://git.drogon.net/wiringPi
Cloning into 'wiringPi'...
fatal: unable to connect to git.drogon.net:
git.drogon.net[0: 188.246.205.22]: errno=Connection refused
```

Changing the installer to use the _unofficial_ GitHub mirror: `https://github.com/WiringPi/WiringPi`

### CODE REVIEW

**Comment by bill-luu** [comment]
Travis build is currently failing

File: src/videoStream.cpp [hunk 1]
```diff
@@ -52,10 +52,25 @@ void VideoStream::signalToQuit()
...
+// The purpose of threading this is because `cap.read()`
+// takes a while to read the camera. And while it's reading
+// from the camera, we can spend time on the main thread calling
+// `RenderCopyEx()` and `RenderPresent()`. This increases framerates.
+//
+// To avoid blocking on the threads using a mutex to read from the `threadImageX`
+// We decided that it'd be worth introducing a small amount of latency, but making it
+// pretty race proof by using triple buffering.
+//
+// In hindsight, http://lazyfoo.net/tutorials/SDL/42_texture_streaming/index.php might
```
Comment (bill-luu) [inline src/videoStream.cpp]: Can we make an issue about this? Just so we remember to follow up

Comment (bill-luu) [reply src/videoStream.cpp]: Also, I would say this comment looks prettier with `/* ..*/` instead of `//`

**Comment by enochtsang** [comment]
the `newfile.txt` needs to be deleted

**Pull request merged**